lib/server.js

```diff
@@ -38,6 +34,18 @@ var TEMPLATE = ['<script id="%s" type="application/javascript">var ',
 
 exports.create = function create(createOptions) {
 
+  // safely require the peer-dependencies
+  var React = util.safeRequire('react');
+  var Router = util.safeRequire('react-router');
```

Comment:
@stevemao if `react-router` is not installed, an error will be thrown on line 39 by default. 

The correct way to do this would be to enclose this in a try/catch and conditionally throw.
condition === `if (!Router && createOptions.routes)`